airflow/providers/amazon/aws/hooks/athena.py

```diff
@@ -91,8 +94,12 @@ def run_query(
         }
         if client_request_token:
             params["ClientRequestToken"] = client_request_token
+        if self.log_query:
```

Comment:
What is the reason why we want to use a flag (`log_query`) to configure whether we log or not? In other words, why dont we log regardless ?

Reply:
see https://github.com/apache/airflow/pull/27917#discussion_r1032945171

Reply:
Oops! I missed that. Thanks for pointing it out. LGTM